File: Python/Algorithm/Greatest.py
maior = 0
quantos = 0
while True:
    valor = int(input())
    if valor < 1:
    	break
    if valor > maior:
        maior = valor
        quantos = 1
    elif valor == maior:
        quantos += 1
print('X =', maior, 'quantos = ', quantos)

#https://pt.stackoverflow.com/q/448608/101

File: JavaScript/Array/Greatest.js
function retornaIndiceMaiorValor() {
    let maior = array[0];
    let indice = 0;
    for (let i = 1; i < array.length; i++) {
        if (array[i] > maior) {
            maior = array[i];
            indice = i;
        }
    }
    return indice;
}

let array = [2, 3, 6, 7, 10, 1];
console.log(retornaIndiceMaiorValor(array));

//https://pt.stackoverflow.com/q/436409/101
